iree/compiler/Dialect/Flow/Transforms/test/dispatch_linalg_on_tensors.mlir

```diff
@@ -50,7 +50,9 @@ func @tensor() -> tensor<2x4xf32> {
   // CHECK-NEXT:       %[[CC:.*]] = flow.dispatch.input.load %[[C]],
   // CHECK-SAME:         offsets = [%[[I]], %[[J]]], sizes = [%[[MIN_I]], %[[MIN_J]]], strides = [%[[C1]], %[[C1]]] :
   // CHECK-SAME:           !flow.dispatch.input<2x4xf32> -> tensor<?x?xf32>
-  // CHECK-NEXT:       %[[RES:.*]] = linalg.matmul {__internal_linalg_transform__ = "workgroup"} ins(%[[AA]], %[[BB]] :
+  // CHECK-NEXT:       %[[RES:.*]] = linalg.matmul
+  // CHECK-SAME:       __internal_linalg_transform__ = "workgroup", iree.codegen.fusion.root_op
```

Comment:
Dont want to check for this attribute. Its a transient in-pass attribute. Why is it even being left on the op?

Reply:
Was there so I can rediscover which op I should use as the anchor to query concrete tiling and distribution scheme. Can remove it and hardcode the list in the following pass.